android/src/main/java/ie/gov/tracing/Tracing.kt

```diff
@@ -111,13 +111,32 @@ object Tracing {
     class BleStatusReceiver : BroadcastReceiver() {
 
         override fun onReceive(context: Context, intent: Intent) {
-
+            var newExposureDisabledReason = Tracing.exposureDisabledReason
             if (BluetoothAdapter.ACTION_STATE_CHANGED == intent.action) {
-                Tracing.getExposureStatus(null)
+                if (isBluetoothAvailable()) {
+                    if (newExposureDisabledReason == "bluetooth") {
+                        newExposureDisabledReason = "starting"
```

Comment:
I'm not sure if `"starting"` is the right value to reset to if bluetooth has been enabled. It seems to be closest to current behaviour where it is the default.

Reply:
We don't want to use `starting`, starting is used for tracking the initialising state and we don;'t want it going back to that state, set is to empty string